grafana/grafana
# Tracing: Send tags, events, and references as raw json

**What this PR does / why we need it**:
Removes unnecessary json parsing on the backend/frontend by sending tags, events, and references as raw json instead of strings. 

**Which issue(s) this PR fixes**:



Relates to #50254

**Special notes for your reviewer**:
To test:
- Run Grafana + Tempo + TNS demo (or similar)
- Query by trace id
- Check that it renders correctly, including attributes, events, and references (if present)

Todo
- Exemplars test fails in CI but passes locally, not sure what's up
